src/Mvc/Mvc.ApiExplorer/src/EndpointParameterDescriptor.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer;
```

Comment:
Nit: we haven't standardized on file-scoped namespaces in the repo yet so we're avoiding adding it for the moment.

Yes, we did sneak some in in the new analyzers project, but 🤪

Reply:
😆 - no worries, I'll change it to the other style.